src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -203,27 +204,11 @@ def create_privacy_notices(
     To avoid any confusing or unexpected behavior, the entire operation is void
     if any of the input data does not satisfy validation criteria.
     """
-    validate_notice_data_uses(privacy_notices, db)
 
-    existing_notices = (
-        PrivacyNotice.query(db).filter(PrivacyNotice.disabled.is_(False)).all()
-    )
-
-    new_notices = [
-        PrivacyNotice(**privacy_notice.dict(exclude_unset=True))
-        for privacy_notice in privacy_notices
-    ]
     try:
-        check_conflicting_data_uses(new_notices, existing_notices)
-    except ValidationError as e:
-        raise HTTPException(HTTP_422_UNPROCESSABLE_ENTITY, detail=e.message)
-
-    return [
-        PrivacyNotice.create(
-            db=db, data=privacy_notice.dict(exclude_unset=True), check_name=False
-        )
-        for privacy_notice in privacy_notices
-    ]
+        return create_privacy_notices_util(db, privacy_notices, PrivacyNotice)  # type: ignore[return-value]
```

Comment:
I move all this into `create_privacy_notices_util` so I can share all this nice validation when populating the templates and then the notices from those templates.